README.md

```diff
@@ -16,36 +19,37 @@ _[Demo and API Docs](https://elements.polymer-project.org/elements/iron-demo-hel
 
 ##&lt;demo-snippet&gt;
 
-
 `demo-snippet` is a helper element that displays the source of a code snippet and
 its rendered demo. It can be used for both native elements and
 Polymer elements.
 
-    Example of a native element demo
+```html
```

Comment:
Ah yeah, that may not have been the intent, but according to Markdown it is part of the code, as any text indented by four or more spaces is code.

https://github.com/PolymerElements/iron-demo-helpers/blob/master/demo-snippet.html#L23